src/nvim/eval.c

```diff
@@ -12251,7 +12255,7 @@ static void f_reltime(typval_T *argvars, typval_T *rettv) FUNC_ATTR_NONNULL_ALL
   // (varnumber_T is defined as int). For all our supported platforms, int's
   // are at least 32-bits wide. So we'll use two 32-bit values to store it.
   union {
-    struct { varnumber_T low, high; } split;
+    struct { uint32_t low, high; } split;
     proftime_T prof;
```

Comment:
Sort of... I think it'd end up swapping the high and low bits when adding them to the list and also check the high bits for overflow instead of the low. But it should convert to and fro `split` and `proftime_T`.